src/module_1/module_1_meteo_api.py

```diff
@@ -1,7 +1,157 @@
-""" This is a dummy example """
+import requests
+import matplotlib.pyplot as plt
+import numpy as np
+import time
+import os
+
+# flake8: noqa E501
+# import time
+# import pandas
+
+API_URL = "https://climate-api.open-meteo.com/v1/climate?"
+COORDINATES = {
+    "Madrid": {"latitude": 40.416775, "longitude": -3.703790},
+    "London": {"latitude": 51.507351, "longitude": -0.127758},
+    "Rio": {"latitude": -22.906847, "longitude": -43.172896},
+}
+VARIABLES = "temperature_2m_mean,precipitation_sum,soil_moisture_0_to_10cm_mean"
+MODELS = (
+    "CMCC_CM2_VHR4,FGOALS_f3_H,HiRAM_SIT_HR," "MRI_AGCM3_2_S,EC_Earth3P_HR,MPI_ESM1_2_XR,NICAM16_8S"
+)
+
+SLEEP_REQUEST = 1
+
+
+DATA_FOLDER_NAME = "src/module_1/Plots"
+if os.path.exists(DATA_FOLDER_NAME) is False:
+    os.makedirs(DATA_FOLDER_NAME)
+
+
+def get_data_meteo_api(city, start_date, end_date):
```

Comment:
For the win : `get_data_meteo_api(city:str , start_date: datetime.date, end_date: datetime.date)`: